src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -622,7 +625,7 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
           case Some(v) => Success(v)
           case _ => Failure(new WdlExpressionException("Unknown error"))
         }
-        val coercedValue = value.flatMap(x => taskInput.wdlType.coerceRawValue(x))
+        val coercedValue = value.map(x => taskInput.wdlType.coerceRawValue(x).get)
```

Comment:
I've been all over WorkflowActor and CallActor with Await removals, but I've thrown away a lot of failed attempts so I'm not really sure.  :frowning: